container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -2096,9 +2096,23 @@ instance_groups:
           "uaa.((KUBERNETES_NAMESPACE)).svc",
           "uaa.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))"
         ]
+      properties.uaadb.address: >
+        ((#DB_EXTERNAL_HOST))
+          ((DB_EXTERNAL_HOST))
+        ((/DB_EXTERNAL_HOST))
+        ((^DB_EXTERNAL_HOST))
+          mysql-proxy-set.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))
+        ((/DB_EXTERNAL_HOST))
```

Comment:
I see this construction in a number of places (Nine, actually), this here the first.
Would it make sense to add an `&anchor` here and have all the other places just `*anchor` reference it ?

Reply:
Thanks.